Project.toml

```diff
@@ -13,9 +13,9 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 ChainRulesCore = "0.9.21"
-ChainRulesTestUtils = "0.5.5"
```

Comment:
we probably don't want to do this, as it blocks us being able to tag a release.
Letting the integration test fail is fine

Reply:
Okay, I've reverted the change. My only worry is that FiniteDifferences v0.12 is now allowed but in no CI tested for compatibility with ChainRules.

Reply:
its true, so might just not do this til we do ChainRulesTestUtils too.
I suspect the PRs are open and waiting for review

Reply:
Let's wait with merging this then until ChainRules runs on some 0.6 of ChainRulesTestUtils.